sre_agent/servers/mcp-server-kubernetes/Dockerfile

```
@@ -16,6 +16,8 @@ RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.
 RUN apt-get update
 RUN apt-get install -y kubectl awscli
 
+RUN apt-get install -y kubectl awscli netcat-openbsd
```

Let's condense this to a single line
```suggestion
RUN apt-get update && apt-get install -y kubectl awscli netcat-openbsd
```